pkgs/development/beam-modules/erlang-ls/default.nix

```diff
@@ -45,4 +46,22 @@ rebar3Relx {
     platforms = platforms.unix;
     license = licenses.asl20;
   };
+  passthru.updateScript = writeScript "update.sh" ''
+    #!/usr/bin/env nix-shell
+    #! nix-shell -i bash -p common-updater-scripts coreutils git gnused gnutar gzip "rebar3WithPlugins { globalPlugins = [ beamPackages.rebar3-nix ]; }"
+
+    set -ox errexit
+    latest=$(list-git-tags https://github.com/${owner}/${repo}.git | sed -n '/[\d\.]\+/p' | sort -V | tail -1)
+    if [ "$latest" != "${version}" ]; then
```

Comment:
small nit, could you use [[ instead of [
it's more bash idiomatic. Let me know if you feel strongly about this or if you want me to go into more details.

Reply:
fixed!